docs/config.rst

```diff
@@ -497,7 +497,8 @@ Complete list of settings that you can put into ``testenv*`` sections:
     .. versionadded:: 3.10
 
     Set to ``true`` if you want virtualenv to upgrade pip/wheel/setuptools to
-    the latest version.
+    the latest version. When this is set you may add e.g. ``VIRTUALENV_PIP=20.3.3``
```

Comment:
I believe this should work without this version too, not?

Reply:
Not for everyone apparently e.g. #1768. That's why I put `you may add`, but maybe that isn't clear?

Regardless if you want any specific version that's not exactly latest you do need the flag.

Reply:
maybe its clearer now?